wiki/Beatmap_Editor/tr.md

```diff
@@ -0,0 +1,44 @@
+# Beatmap Düzenleyici
+
+<!-- TODO: needs to be revisited when the articles under Beatmap editor no longer match what's written here -->
+
+![A screenshot of the editor's compose tab showing pishifat's Donor Song beatmap](img/compose.jpg)
+
+*Beatmap düzenleyici kısayolları için, bakınız: [Kısayol referansı & Beatmap düzenleyici](/wiki/Shortcut_key_reference#beatmap-editor)*
+
+**Beatmap düzenleyici** osu!'nun oyun içi [beatmap](/wiki/Beatmaps) yapma bölümüdür. [Beatmapping](/wiki/Beatmapping) osu! için beatmap yapma işidir
+
+Beatmap düzenleyici 4 ana kısma ayrılmıştır (sol üstte görebilirsiniz):
+
+- Compose (Oluştur)
+- Design (Dizayn)
+- Timing (Zamanlama)
+- Song Setup (Müzik Kurulumu)
+
+## Compose (Oluştur)
+
+[Compose (Oluştur)](/wiki/Beatmap_Editor/Compose) sekmesi tüm [vuruş objelerinin](/wiki/Hit_Objects) ve oynayış ile alakalı diğer elemanların oluşturulduğu yerdir. Pek çok mapper vaktinin çoğunu bu sekmede geçirmesi bu yüzdendir.
```

Comment:
```suggestion
[Compose (Oluştur)](/wiki/Beatmap_Editor/Compose) sekmesi tüm [vuruş objelerinin](/wiki/Hit_Objects) ve oynayış ile alakalı diğer elemanların oluşturulduğu yerdir. Pek çok mapper'ın vaktinin çoğunu bu sekmede geçirmesi bu yüzdendir.
```
fix typo

Reply:
Well Im new on this Contributing and PR things I dont really know this system well. But thanks for your recommendations. zeusminus and Lxran.